sprokit/processes/core/handle_descriptor_request_process.cxx

```diff
@@ -1,21 +1,42 @@
-// This file is part of KWIVER, and is distributed under the
-// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
-// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+/*ckwg +29
```

Comment:
Do we want to leave the short copyright header here?

Reply:
Yes we do (especially as the date is not even right in the long versions). Not sure what happened here?

Reply:
I'm sorry, this was my error and I should have addressed it sooner. I previously merged `master` into the feature branch on #1157. We decided to break that PR up and because commits touched a bunch of files, I just copied files over onto new branch. So it introduced regressions from the outdated `master`. 

Before more work is put into this branch, I'd like to assess whether these changes are actually needed for the BurnOut migration. That might not become evident until I begin integration work, or @mattdawkins may know.